src/SDL_rtf.c

```diff
@@ -121,6 +121,10 @@ int RTF_Load_RW(RTF_Context *ctx, SDL_RWops *src, int freesrc)
             RTF_SetError("Couldn't find font for text");
             retval = -1;
             break;
+        default:
+            RTF_SetError("Unknown error");
+            retval = -1;
```

Comment:
This should go into SDL2 branch too

Reply:
80d8cd0f6bc360bdf90c2704c1533257f31cfb8c